pom.xml

```diff
@@ -809,7 +809,7 @@
     <tar.name>${project.build.finalName}.tar.gz</tar.name>
     <maven.build.timestamp.format>yyyy-MM-dd'T'HH:mm</maven.build.timestamp.format>
     <buildDate>${maven.build.timestamp}</buildDate>
-    <compileSource>1.8</compileSource>
+    <compileSource>17</compileSource>
     <releaseTarget>8</releaseTarget>
```

Comment:
What does it mean to have compile target at 17 but release target at 8? Can javac emit bytecode for 17 features for an 8 VM?

We are dropping JDK8 runtime support, so I think we can move the release target up as well.

Reply:
ReleaseTarget 17 will break mockito, need to dig more...